tests/test_web_app_steps.py

```diff
@@ -31,7 +31,6 @@ def setUp(self):
         self.app_context.report = Mock()
         data_store.spec[app_context_key] = self.app_context
         self.element = Mock()
-        os.environ["driver_implicit_timeout"] = '0'
```

Comment:
I think this timeout existed here because of unnecessarily long test execution time. Better to leave it.